ktor-server/ktor-server-plugins/ktor-server-swagger/jvm/src/io/ktor/server/swagger/SwaggerConfig.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2014-2022 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.server.swagger
+
+/**
+ * Configuration for Swagger UI endpoint.
+ */
+public class SwaggerConfig {
+    internal var customStyle: String? = null
+
+    /**
+     * Specify Swagger UI version to use.
+     */
+    public var version: String = "4.14.0"
+
+    /**
+     * Set url for custom css style for swagger-ui.
+     *
+     * You can try: https://unpkg.com/swagger-ui-themes@3.0.1/themes/3.x/theme-monokai.css
+     */
+    public fun customStyle(path: String?) {
```

Comment:
Just in case here which customizations swagger has:
https://swagger.io/docs/open-source-tools/swagger-ui/usage/configuration/

Reply:
any idea on this?

Reply:
Let me check

Reply:
It looks quite generic, so I'm not sure how it should be done. Could you tell me if you have any ideas?

Reply:
passing json object, or path (url or filesystem) to configuration file is also an option.
At least for me, changing `layout` and `validationUrl` were needed in the past